Description: Fix annoying "Format a4 redefined" messages
 According to the openoffice docs on the excel file format, 0xa4 is the first
 user-defined format, so it doesn't make sense to map it to a fixed strftime()
 format: https://www.openoffice.org/sc/excelfileformat.pdf
Author: Olly Betts <olly@survex.com>
Forwarded: no
Last-Update: 2015-08-27

--- a/src/xlsparse.c
+++ b/src/xlsparse.c
@@ -551,8 +551,10 @@
 	} else 	
 		if ((index>=0x2d) && (index<=0x2F)) {
 			return offset+index-0x2d+9;
+#if 0 /* 0xa4 is the first user defined format */
 		} else if (index==0xa4) {	
 			return 12+offset;
+#endif
 		} else 	
 			return 0;
 }	
@@ -561,7 +563,7 @@
  * GetBuiltInDateFormat stores and returns
  * built in xls2csv strftime formats.
  */
-#define NUMOFDATEFORMATS 13
+#define NUMOFDATEFORMATS 12
 char *GetBuiltInDateFormat(int dateindex) {
 	static char *formats[]={
 		/* reserved  */ NULL, /* BuiltInDateFormatIdx use dateindex=0 as flag format not found */
@@ -577,7 +579,9 @@
 		/* 0x2d */ "%M:%S",		/* 10 */
 		/* 0x2e */ "%H:%M:%S",		/* 11 */
 		/* 0x2f */ "%M:%S",		/* 12 */
+#if 0
 		/* 0xa4 */ "%m.%d.%Y %l:%M:%S %p"	/* 13 */
+#endif
 	};
 	if (dateindex>0 && dateindex <= NUMOFDATEFORMATS) {
 	  return formats[dateindex];
